components/network/tor/Makefile

```diff
@@ -63,12 +57,22 @@ build:		$(BUILD_64)
 
 install:	$(INSTALL_64)
 
+# One test failed:
+# error: util/map_anon_nofork:
+# FAIL ~/oi-userland/components/network/tor/tor-0.4.1.6/src/test/test_util.c:6223: assert(buf[0] OP_EQ 0xd0): -48 vs 208
+#  [map_anon_nofork FAILED]
+# see: https://trac.torproject.org/projects/tor/ticket/31897
 test:		$(TEST_64)
```

Comment:
IMO install: and test: targets aren't needed anymore because they are included in common.mk.